### DISCUSSION TITLE
About wireless support

### DISCUSSION THREAD

**Discussion by Blae1027** [brndnmtthws/conky#2061] [Q&A] [answered]
I’ve just noticed (in Fedora 40) that none of the ‘wireless_*’ variables are recognized in conky.

And I know that the wlan support was disabled in the Fedora conky package due to this change:
https://fedoraproject.org/wiki/Changes/RemoveWirelessExtensions

So is this feature likely to be added in the future?

````

conky 1.19.6_pre compiled 2024-02-01 for Linux x86_64

Compiled in features:

System config file: /etc/conky/conky.conf
Package library path: /usr/lib64/conky


 General:
  * math
  * hddtemp
  * portmon
  * IPv6
  * Curl
  * RSS
  * support for IBM/Lenovo notebooks
  * old configuration syntax
  * Imlib2
  * OSS mixer support
  * apcupsd
  * iostats
  * ncurses
  * Internationalization support

 Lua bindings:
  * Cairo
  * Imlib2
 X11:
  * Xdamage extension
  * Xinerama extension (virtual display)
  * Xshape extension (click through)
  * XDBE (double buffer extension)
  * Xft
  * ARGB visual
  * Own window
  * Mouse evenets

 Music detection:
  * CMUS
  * MPD

 Default values:
  * Netdevice: eno1
  * Local configfile: $HOME/.conkyrc
  * Localedir: /usr/share/locale
  * Maximum netdevices: 256
  * Maximum text size: 16384
  * Size text buffer: 256

`````

**Comment by Caellian** [comment, marked as answer]
Yeah, it's disabled on fedora because it used an old network interface that's been dropped. I'm working on supporting the new one in #1879, and the related issue is #1001.

**Reply by Blae1027** [reply]
Thanks for your reply. Well, I can only think that this work is too hard...

**Reply by Caellian** [reply]
The main problem is that there's very little (unfinished) documentation on how to use the new interface, so it takes a lot of time to try out various things until one that works is found.

**Reply by Blae1027** [reply]
Anyway, thank you for the work and I look forward to the next release
